driver-core/src/test/unit/com/mongodb/operation/OperationUnitSpecification.groovy

```diff
@@ -106,12 +106,12 @@ class OperationUnitSpecification extends Specification {
 
         if (checkCommand) {
             1 * connection.command(*_) >> {
-                it[1] == expectedCommand
+                assert(it[1] == expectedCommand)
```

Comment:
Good catch.  What `assert` is being called here?  It is Spock's?

Reply:
Groovy / Java assert